hummingbot/market/coinbase_pro/coinbase_pro_market.pyx

```diff
@@ -522,11 +483,26 @@ cdef class CoinbaseProMarket(MarketBase):
             int64_t current_tick = <int64_t>(self._current_timestamp / 60.0)
         if current_tick > last_tick or len(self._trading_rules) <= 0:
             product_info = await self._api_request("get", path_url="/products")
-            trading_rules_list = TradingRule.parse_exchange_info(product_info)
+            trading_rules_list = self._format_trading_rules(product_info)
             self._trading_rules.clear()
             for trading_rule in trading_rules_list:
                 self._trading_rules[trading_rule.symbol] = trading_rule
 
+    def _format_trading_rules(self, trading_rules: List[Any]) -> List[TradingRule]:
```

Comment:
rename trading_rules to raw_trading_rules?

Reply:
done